### PR TITLE
[139] Product Page using HTML

### PR DESCRIPTION
**Pull request by olakanmigoodnessb**
Capture 1
Capture 2
Capture 3
Capture 4

### CODE REVIEW

File: submissions/goodnessOlakanmi/HTML/Product Page.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+	<section>
+		<h2>Shop for similar products</h2>
+	<ul class="other-products">
+		<li><a href="#">Zebra Print Slippers</li>
+		<li><a href="#">Gold Slippers</li>
+		<li><a href="#">Fur Slides</li>
+	</ul>
+	</section>
+
+
```
Comment (Ireti203) [inline submissions/goodnessOlakanmi/HTML/Product Page.html:69]: Remove whitespaces

File: submissions/goodnessOlakanmi/HTML/Product Page.html [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+	<footer>
+		<ul class="soc-media">
+			<li><a href="#" class="fa fa-https://www.twitter.com">Twitter</a></li>
+			<li><a href="#" class="fa fa-https://www.facebook.com">Facebook</a></li>
+			<li><a href="#" class="fa fa-https://www.pinterest.com">Pinterest</a></li>
+		</ul>
+
```
Comment (Ireti203) [inline submissions/goodnessOlakanmi/HTML/Product Page.html:77]: Remove whitespace

**Review by Ireti203 — COMMENTED** [review]
You have two product.html file. Kindly remove the unused one.

**Comment by olakanmigoodnessb** [comment]
> You have two product.html file. Kindly remove the unused one.

I thought so too, bit I just don't know how to remove it.

File: submissions/goodnessOlakanmi/HTML/Product Page.html (hunk 1 above)
Comment (olakanmigoodnessb) [reply submissions/goodnessOlakanmi/HTML/Product Page.html]: I have removed the whitespaces

**Comment by olakanmigoodnessb** [comment]
Hi, good evening. Sorry to bother you. After I couldn't sort out removing
duplicate html file, I closed the entire pull request and opened another
one altogether.  It's been about 10 hours since I did that and you are yet
to to review it. Please do, thanks. I really don't want to miss the be
behind the deadline.

PS: I really do appreciate all the effort you put into guiding us in this
program.

**Pull request closed without merging**